### DISCUSSION TITLE
calibrage iu22 pour activer l'écran tactile

### DISCUSSION THREAD

**Discussion by CIZAMugaruka** [PlusToolkit/PlusLib#1201] [Q&A]
hello, I can have the software to generate the iu22 parameter codes, my touch screen does not work, or cite it to generate the code

**Comment by lassoan** [comment]
Interface to Philips 3D TEE probes in Plus is available here: https://github.com/PlusToolkit/PlusLib/tree/master/src/PlusDataCollection/Philips3D

You need to contact Philips to get access to their streaming interface.
